test/function/samples/check-exports-exportedBindings-when-export-all-source/main.js

```diff
@@ -0,0 +1,2 @@
+export * from './module.js';
+assert.ok(true);
```

Comment:
Actually in this case, you do not need the `assert.ok(true)` because it is an entry point with exports, those exports will not be removed.

Reply:
But I ran into a test error `Generated an empty chunk: "main"` before adding `assert.ok(true)`, see 
https://github.com/rollup/rollup/actions/runs/3590917035/jobs/6044842787

Reply:
Fair enough